packages/office-ui-fabric-react/src/components/GroupedList/examples/GroupedList.CustomCheckbox.Example.tsx

```diff
@@ -39,19 +42,20 @@ export const GroupedListCustomCheckboxExample: React.FunctionComponent = () => {
         }),
       ),
   );
-  const selection = useConst(() => new Selection({ items }));
+  const selection = (useConst(() => new Selection({ items })) as unknown) as ISelection<IObjectWithKey>;
```

Comment:
Curious why this was needed? Seems like it ought to be inferred correctly, or if not maybe there's some way to specify a generic to get the proper result by default.

Reply:
It is definitely not inferred correctly. I'll fiddle a bit more with it.